src/main/java/scheduler_use_case/SchedulerController.java

```diff
@@ -0,0 +1,32 @@
+package scheduler_use_case;
+
+import entities.StudentUser;
+import entities.Task;
+
+import java.util.ArrayList;
+
+public class SchedulerController {
```

Comment:
I think this file should be in the screens package instead of the use cases but other than that everything else looks good!